subscription_oca/data/sale_subscription_data.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+  <data>
+    <record id="seq_id" model="ir.sequence">
+      <field name="name">sale_subscription_sequencer</field>
+      <field name="code">sale.subscription</field>
+      <field name="prefix">SUB</field>
+      <field name="padding">5</field>
+    </record>
+  </data>
+
+  <data noupdate="1">
+    <record id="subscription_stage_draft" model="sale.subscription.stage">
+      <!-- <field eval="True" name="active"/> -->
+      <field name="name">Ready to start</field>
+      <field name="sequence">0</field>
+      <field name="type">pre</field>
+      <field name="description">
+        DEFAULT STAGE
```

Comment:
is this "DEFAULT STAGE" wanted?
Also, wouldn't be better to flag them as `noupdate`?

Reply:
ahh sorry, I've looked only at the top :sweat: 

> is this "DEFAULT STAGE" wanted?

I mean: why do you need this sentence in the description?

Reply:
Ok, I delete it